jax_md/io/io.py

```diff
@@ -0,0 +1,33 @@
+def vec2str(x):
+  """Convert a vector to string.
+  """
+  return "\t".join([str(i) for i in x])
```

Comment:
Can we call local functions `_vec2str` or better yet `_xyz_vec2str` since this is only true for the xyz file format.